modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/warrior.dm

```diff
@@ -82,5 +85,8 @@
 	agility_target.remove_movespeed_modifier(/datum/movespeed_modifier/warrior_agility)
 	agility_target.unable_to_use_abilities = FALSE
 
+	agility_target.melee_damage_lower = 30
```

Comment:
I would suggest to use `initial()` proc, if it is applicable

Reply:
oh right that exists